doc/usermanual.md

```diff
@@ -609,78 +609,48 @@ Stores the information about the IoTAgent for further use in the `retrieveVersio
 All contributions to this project are welcome. Developers planning to contribute should follow the [Contribution Guidelines](./docs/contribution.md)
 
 ### Project build
-The project is managed using Grunt Task Runner.
+The project is managed using npm.
 
 For a list of available task, type
 
 ```bash
-grunt --help
+npm run
 ```
 
 The following sections show the available options in detail.
 
 ### Testing
-[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Chai](http://chaijs.com/) Assertion Library + [Sinon](http://sinonjs.org/) Spies, stubs.
+[Mocha](http://visionmedia.github.io/mocha/) Test Runner + [Should.js](https://shouldjs.github.io/) Assertion Library.
 
-The test environment is preconfigured to run [BDD](http://chaijs.com/api/bdd/) testing style with
-`chai.expect` and `chai.should()` available globally while executing tests, as well as the [Sinon-Chai](http://chaijs.com/plugins/sinon-chai) plugin.
+The test environment is preconfigured to run BDD testing style.
 
 Module mocking during testing can be done with [proxyquire](https://github.com/thlorenz/proxyquire)
 
 To run tests, type
 
 ```bash
-grunt test
-```
-
-Tests reports can be used together with Jenkins to monitor project quality metrics by means of TAP or XUnit plugins.
-To generate TAP report in `report/test/unit_tests.tap`, type
-
-```bash
-grunt test-report
+npm test
```

Comment:
"npm test" or "npm run test" ?

Reply:
(Unsure, just to confirm)

Reply:
npm supports some `scripts` by default: test, start, stop, version. These scripts can be run directly with `npm <stage>`.

Additionally, arbitrary scripts can be executed by running `npm run-script <stage>` or `npm run <stage>`

Reply:
I didn't know :) Thanks for the clarification!

NTC